terraform/meta/main.tf

```diff
@@ -122,7 +122,38 @@ data "aws_caller_identity" "current" {}
 module "github_actions_access" {
   source = "../modules/github_actions_access"
 
-  application_repo = "communitiesuk/submit-social-housing-lettings-and-sales-data"
-  ecr_arn          = module.ecr.repository_arn
-  meta_account_id  = data.aws_caller_identity.current.account_id
+  meta_account_id = data.aws_caller_identity.current.account_id
+  repositories = {
+    application = {
+      name = "communitiesuk/submit-social-housing-lettings-and-sales-data",
+      policies = [
+        { key = "push_ecr_images", arn = module.ecr.push_images_policy_arn },
+        { key = "access_non_prod_state", arn = module.non_prod_backend.state_access_policy_arn }
+      ]
+    },
+    infrastructure = {

```

Comment:
N.B. Right now this isn't actually necessary, but I'd got so close before the move to have the app repo run this bit that it seemed worth leaving in, given we're expecting to set up automated terraform deployments at least for the dev environment (hopefully before too long!)